pkg/controller/storagecluster/stork.go

```diff
@@ -635,6 +636,10 @@ func (c *Controller) getStorkDeploymentSpec(
 		)
 	}
 
+	if cluster.Spec.Stork.HostNetwork != nil {
```

Comment:
If that is nil (ie been removed from the storagecluster spec) we should reset it to `false` in the deployment

Reply:
The default value of deployment's hostNetwork is false. On line 523 when we detect the previous value does not match current (nil), we redeploy the spec. It will result in having a deployment spec with host network as false (default). 
Will update the test to cover `true -> nil` scenario.